lib/dev.ts

```diff
@@ -106,10 +126,12 @@ export async function devMulti(root: string, srcs: string[], dest: string, opts:
     appDevJsons.push(devJson);
   }
 
+	const gatewayObject: GatewayConfigObject = buildGatewayObject(opts.gateway, {})
```

Comment:
We can remove this from devMulti -- it gets complicated fast. Let's only do a single dev app.

devMulti currently used DEFAULT_LOCAL_GATEWAY_URL and we should keep it that way.